### PR TITLE
Add governance model; update roadmap

### PR DESCRIPTION
**Pull request by drammock**
- adds documentation of our governance model (incl. formalizing the notion of "core devs" as a Steering Council)
- adds names / institution logos for contributors who meet the Steering Council threshold
- updates our roamap

@britta-wstnr LMK if you want Donders listed in addition to Aarhus

cc @mmagnuski @sappelhoff @bloyl @rob-luke @GuillaumeFavelier @hoechenberger @jasmainak @cbrnr @dengemann @wmvanvliet 

FYI this is closely modeled on governance docs used by Jupyter, NumPy, and SciPy. That's not a guarantee that it's what we want, but it seems like a sensible starting place.

### CODE REVIEW

**Comment by cbrnr** [comment]
Very nice! We should try to get more diverse, but we're all aware of this I guess.

**Comment by drammock** [comment]
Shortcut to renderings:

- [new logos on homepage](https://28346-1301584-gh.circle-artifacts.com/0/dev/index.html) (SWPS, MPIB, CHOP, Macquarie)
- [list of institutional partners](https://28346-1301584-gh.circle-artifacts.com/0/dev/funding.html#institutional-partners) on the funding page
- [governance page](https://28346-1301584-gh.circle-artifacts.com/0/dev/overview/governance.html)
- [steering council page](https://28346-1301584-gh.circle-artifacts.com/0/dev/overview/people.html#governance-people)
- [roadmap](https://28346-1301584-gh.circle-artifacts.com/0/dev/overview/roadmap.html)

File: doc/overview/people.rst
Comment (sappelhoff) [inline doc/overview/people.rst:30]: would it be sensible to have these names as links, similar to how we do it in the Changelog? Simply by turning each list item into an RST link, and then including https://github.com/mne-tools/mne-python/blob/e0ec4a0de8d9722f2f8488797bafdf9334cb4b0a/doc/changes/names.inc#L1 somewhere

File: doc/overview/roadmap.rst
Comment (GuillaumeFavelier) [inline doc/overview/roadmap.rst:161]: I think it would be more accurate to say:

```suggestion
1. *Proper notebook support (through ipyvtklink)* (complete)
```

A few versions ago, `panel` was the backend used for notebook support but we switched to `ipyvtk` since then (which is based on `ipycanvas`). Sorry for being late to notice this one :pray:

**Review by agramfort — APPROVED** [review]
I think it's important before we merge that all members of the steering council approve this PR:

- [x] @britta-wstnr 
- [x] @cbrnr 
- [x] @drammock (can't approve own PR but it's implied)
- [ ] @dengemann 
- [x] @GuillaumeFavelier 
- [x] @larsoner
- [ ] @bloyl 
- [x] @jasmainak 
- [x] @wmvanvliet 
- [x] @mmagnuski 
- [ ] @hoechenberger 
- [x] @rob-luke 
- [x] @sappelhoff

**Review by wmvanvliet — APPROVED** [review]

**Review by cbrnr — APPROVED** [review]

**Comment by mmagnuski** [comment]
Looks great!

**Review by mmagnuski — APPROVED** [review]

**Review by rob-luke — APPROVED** [review]

**Review by larsoner — APPROVED** [review]

**Comment by cbrnr** [comment]
@agramfort I've requested reviews by those people so that they will receive a notification.

File: doc/overview/governance.rst
Comment (britta-wstnr) [inline doc/overview/governance.rst:58]: `by their receiving “commit rights” to the Project GitHub` sounds a bit awkward to me - as a non-native English speaker I stumbled across it while reading. Maybe there is a better way to phrase it?

**Review by britta-wstnr — APPROVED** [review]
After a quick discussion with @drammock we will skip adding the Donders in this PR (but I will see if we should add them later).

Apart from the small comment above, LGTM!

File: doc/overview/governance.rst
Comment (drammock) [reply doc/overview/governance.rst:58]:
```suggestion
Traditionally, Project leadership was provided by a subset of Contributors,
informally called Core Developers, whose active and consistent contributions
were rewarded by granting them “commit rights” to the Project GitHub
```
is this better @britta-wstnr ?

Comment (drammock) [inline doc/overview/governance.rst:58]:
```suggestion
Traditionally, Project leadership was provided by a subset of Contributors,
informally called Core Developers, whose active and consistent contributions
were rewarded by granting them “commit rights” to the Project GitHub
```
better?

**Review by hoechenberger — APPROVED** [review]
Thank for working on this, @drammock! Looking good to me!

**Comment by dengemann** [comment]
Thank you for pulling this off — it looks great! I am wondering to which extent we shall also subsume other satellite projects in the MNE ecosystem under this governance, or at least mention them. Thinking of the bids pipeline, autoreject, mne-hcp, mne-r, mne-nirs, etc.

**Comment by sappelhoff** [comment]
> Thank you for pulling this off — it looks great! I am wondering to which extent we shall also subsume other satellite projects in the MNE ecosystem under this governance, or at least mention them. Thinking of the bids pipeline, autoreject, mne-hcp, mne-r, mne-nirs, etc.

good point.

At least for mne-bids and bids-pipeline I think it'd be reasonable to say that these fall under the same governance.

- they are in the same GH organization
- Alex is heavily involved in both (fitting to his BDFL role for MNE-Python)
- the here suggested steering council members are maintainers at these packages (Alex, Richard, Mainak, myself), and other steering council members have contributed to them in the past (at least for mne-bids)

I cannot speak for the other packages, but it'd be good to have a policy at least for the projects under the mne-tools org.

**Comment by agramfort** [comment]
we had discussed this with @larsoner @drammock and we decide for now to
focus on mne-python

>

**Comment by larsoner** [comment]
> > At least for mne-bids and mne-bids-pipeline I think it'd be reasonable to say that these fall under the same governance.

> we had discussed this with @larsoner @drammock and we decide for now to focus on mne-python

I would suggest that other repos under the mne-tools umbrella open short PRs that basically say something like:
```
Governance
==========
We follow the same governance model as MNE-Python described :ref:`here <mne:governance>`,
with the exception that the Steering Council and institutional partners differ, see below.

Steering Council
----------------
...

Institutional Partners
----------------------
...
```
This has the advantage that changes relevant to a given repo's structure are handled by that repo rather than at the `mne-python` repo level, but we can all share the same effective governance model with minimal extra work.

Once this is added, we could also link from the MNE-Python governance doc (here) to those of the other repos in a `See Also` sort of section.

**Comment by dengemann** [comment]
@larsoner that sounds good. I also had in mind perhaps making clear that the mne-umbreall repositories buy into the same values as MNE and are maintained by subsets of MNE steering council members, etc. We could then link those project in the roadmap file or so as was already done for men-bids.

**Comment by larsoner** [comment]
> I also had in mind perhaps making clear that the mne-umbreall repositories buy into the same values as MNE and are maintained by subsets of MNE steering council members, etc. We could then link those project in the roadmap file or so as was already done for men-bids.

I'm not sure that's true in all cases, though, hence (another reason, in addition to separation-of-duties) why I think it should be up to the individual projects to state this rather than making a blanket statement in MNE-Python's doc. For example, `mne-tools/mne-cpp` has a fairly separate steering committee (I think it's actually no overlap?) and governance model (at the very least I don't think Alex is BDFL there, and they might not even follow BDFL). There might be others, too. For example, in `mne-r` @dengemann you might be considered BDFL for example, etc.

Maybe we could say something less concerete in the See Also intro, though, like:
```
See Also
--------
Many projects in the `mne-tools` organization follow a similar governance structure
to MNE-Python. For more information, see their respective governance documents:

- :ref:`MNE-BIDS <mne-bids:governance>`
- ...
```

Also @dengemann any chance you can explicitly "approve" this PR? Would be nice to have all the "requested reviews" checked off according to GitHub here.

**Comment by rob-luke** [comment]
> making clear that the mne-umbreall repositories buy into the same values as MNE and are maintained by subsets of MNE steering council members

This was my understanding when I moved MNE-NIRS in to the organisation (although that was a complete assumption by me at the time). Happy to add whatever boilerplate text is decided on, or just to have a statement in MNE-python outlining the fact.

**Review by dengemann — APPROVED** [review]

**Comment by agramfort** [comment]
we need a final approval from @bloyl

**Comment by larsoner** [comment]
@bloyl I'm going to go ahead and merge but we'd still appreciate an approval (I think you can still give it after merge!) and are happy to hear and thoughts/comments/change requests

**Review by bloyl — COMMENTED** [review]
I think this great and appreciate the effort in formalizing everything.

**Comment by larsoner** [comment]
Awesome! Can you click the official "Approve" button? That would complete the set

**Comment by bloyl** [comment]
Sorry for cycling through to this so late. 
I think it looks great and appreciate being included.

**Comment by larsoner** [comment]
... just realized that you can't "approve" once it's merged. Never mind!

**Comment by bloyl** [comment]
yeah I was hoping it would show if I left a comment :) but it didn't

**Pull request merged**
